hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +48,52 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB Mass Storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+.
+
+## Raspberry Pi 3B+
+
+The Raspberry Pi 3B+ supports USB Mass Storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
```

Comment:
...and the CM3+ ?

Reply:
Thank you; I've added the CM3+.

Initially I started this update to add my experiences on the RPI 3B+ working out of the box; which wasn't clearly reflected in this documentation (in my opinion). I have no experience with the CM3+ but given it's similarity in hardware; I can assume it will work?